File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task113_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <string>

int main() {
    int numStrings;
    std::cout << "Enter the number of strings: ";
    std::cin >> numStrings;
    std::string result = "";
    for (int i = 0; i < numStrings; i++) {
        std::string temp;
        std::cout << "Enter string " << (i + 1) << ": ";
        std::cin >> temp;
        result += temp + " ";
    }
    std::cout << "Concatenated string: " << result << std::endl;
    return 0;
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task113_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number of strings:");
        int n = scanner.nextInt();
        String result = "";
        for (int i = 0; i < n; i++) {
            System.out.println("Enter string " + (i + 1) + ":");
            result += scanner.next();
        }
        System.out.println("Concatenated string: " + result);
    }
}